### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Kayonga99**
## The following were implemented:

- No Linter errors.
- Use correct git-flow.
- Set up a new project with webpack.
- Create an index.html.
- Create an index.js.
- Create an index.css.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @Kayonga99 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

#### You have submitted a project that has no linters checks.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+<h1>Today's To Do</h1>
+<i class="fa-solid fa-rotate fa-lg font-awesome-icon"></i>
+</div>
+<div>
+<input placeholder="Add to your list...">
+</div>`;
 
-  return element;
-}
+taskList.forEach((e) => {
```
Comment (SajjadAhmad14) [inline src/index.js]: Please define a function that iterates over the tasks array and populates the HTML list item element for each task.

Moreover, On page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the values of the `index`  property for each task. Kindly see the screenshot below for reference.
vs_pic

list

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi KC, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -39,27 +32,48 @@ The browser will open and you should see this text : "Hello webpack!", if this i
...
 
-[click here!](https://webpack.js.org/guides/getting-started/)
+npx stylelint "\*_/_.{css,scss}"
+npx hint .
+npx eslint .
 
-## Show your support
+- Web Browser
+- [Visual Studio code](https://code.visualstudio.com/)
+[Live demo]()
```
Comment (elmar8287) [inline README.md:58]: - [ ] [OPTIONAL] Please, consider fixing the broken live demo link or just delete this link on current milestone 👍

File: README.md [hunk 3]
```diff
@@ -39,27 +32,48 @@ The browser will open and you should see this text : "Hello webpack!", if this i
...
-- Inspiration
-- etc
+
+## Show your support
+
+Give a ⭐️ if you like this project!
 
 ## 📝 License
 
 This project is [MIT](./MIT.md) licensed.
```
Comment (elmar8287) [inline README.md:75]: - [x] [OPTIONAL] Please, consider personalizing the MIT.md file, by modifying your and app name 👍

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- You have applied all suggestions from the previous review on this milestone 🥇 

### Optional suggestions

- Please, consider my comments under the review 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. The previous reviewer missed some of them, and addressing them will improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**